ga4gh/backends.py

```diff
@@ -411,20 +416,20 @@ def searchVariantSets(self, request):
         if request.pageSize is None:
             request.pageSize = 100
         response = protocol.GASearchVariantSetsResponse()
-        j = 0
+        currentIndex = 0
         if request.pageToken is not None:
-            j = int(request.pageToken)
-        v = []
-        while j < len(self._variantSetIds) and len(v) < request.pageSize:
-            vs = protocol.GAVariantSet()
-            vs.id = self._variantSetIds[j]
-            vs.datasetId = "NotImplemented"
-            vs.metadata = self._variantSets[vs.id].getMetadata()
-            v.append(vs)
-            j += 1
-        if j < len(self._variantSetIds):
-            response.nextPageToken = j
-        response.variantSets = v
+            currentIndex = int(request.pageToken)
+        variantSetList = []
+        while currentIndex < len(self._variantSetIds) and len(variantSetList) < request.pageSize:
+            variantSet = protocol.GAVariantSet()
+            variantSet.id = self._variantSetIds[currentIndex]
+            variantSet.datasetId = self._dataDir
```

Comment:
Is this a good idea? dataDir might contain an arbitrary file system path, which I don't think we should leak. Plus it's not very good as an ID. Maybe we should just have an attribute ``self._id`` which is derived when we instantiate the VariantSet? This is probably outside the context of this PR, so perhaps it's better to revert to the static string for now.